server/repository/repository.go

```diff
@@ -82,7 +82,16 @@ func (s *Server) List(ctx context.Context, q *repositorypkg.RepoQuery) (*appsv1.
 	items := make([]appsv1.Repository, 0)
 	for _, url := range urls {
 		if s.enf.Enforce(ctx.Value("claims"), rbacpolicy.ResourceRepositories, rbacpolicy.ActionGet, url) {
-			items = append(items, appsv1.Repository{Repo: url})
+			repo, err := s.db.GetRepository(ctx, url)
+			if err != nil {
+				return nil, err
+			}
+			username := repo.Username
+			if username == "" {
+				username = "-"
```

Comment:
um, can I ask what this is please?

Reply:
This is supposed to have the username actually displayed in CLI's "repo list" function (the username never made its way back to the client).

I already moved the display logic to the client, but not pushed the change to my branch yet.